.github/workflows/ci.yml

```diff
@@ -15,7 +15,6 @@ env:
   RUN: ${{ github.run_id }}-${{ github.run_number }}
 
 jobs:
-
```

Comment:
We should add a Github action to run rubocop!
Here's one I found: https://github.com/marketplace/actions/rubocop-linter-action
We could do this in a followup CR

Reply:
Agreed - I'd like to add that as a check. There are currently a few rubocop violations that couldn't be auto-corrected.  I was waiting for you're refactor to move things around before fixing them + adding a github action.